src/main/java/org/assertj/core/error/OptionalShouldBeEmpty.java

```diff
@@ -23,18 +24,48 @@
  */
 public class OptionalShouldBeEmpty extends BasicErrorMessageFactory {
 
-  private OptionalShouldBeEmpty(Object optionalValue) {
-	super("%nExpecting an empty Optional but was containing value: <%s>.", optionalValue);
-  }
+    private OptionalShouldBeEmpty(Object optionalValue) {
```

Comment:
add a `String` parameter to get the `Optional` type to have more precise error message:
```
"Expecting an empty %s but was containing value: <%s>."
```
That should give for `OptionalDouble`: 
```
"Expecting an empty OptionalDouble but was containing value: <1.0>."
```